sr_multi_moveit/sr_box_ur10_moveit_config/launch/trajectory_execution.launch.xml

```diff
@@ -11,6 +11,7 @@
   <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
   <!-- Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling) -->
   <param name="trajectory_execution/allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <param name="trajectory_execution/allowed_start_tolerance" value="0.1"/> <!-- default 0.01 -->
```

Comment:
if this is needed for the test only I would set it directly there in the code or in the test launch file, so it won't affect the real robot and sim.

Reply:
Added as a param